static/js/main.js

```diff
@@ -107,25 +112,26 @@ var ep = window['ep'] || {
             // container is being opened
             var locked_by = lock_container.text();
             if (locked_by && locked_by != ep._username &&
-                locked_by != ep._username_alias &&
-                confirm("This module is used by " + locked_by + ". Do you " +
+                locked_by != ep._username_alias) {
```

Comment:
I don't get reason for this change, and it actually breaks code.
Now, if user does not confirm force unlock , you still get to L129 and mark module as used